lib/rspec/core/formatters/exception_presenter.rb

```diff
@@ -31,8 +31,31 @@ def colorized_message_lines(colorizer=::RSpec::Core::Formatters::ConsoleCodes)
           end
         end
 
-        def formatted_backtrace
-          backtrace_formatter.format_backtrace(exception_backtrace, example.metadata)
+        def formatted_backtrace(exception = @exception)
+          lines = backtrace_formatter.format_backtrace((exception.backtrace || []), example.metadata)
+          potential_lines = []
+
+          while exception.respond_to?(:cause) && exception.cause
```

Comment:
>you can pin to the exception_cause_supported branch on rspec-support for the time being.

@JonRowe not sure on how to pin a branch from the `rspec-support` repo? I've setup my machine for rspec development by cloning `rspec-dev` and following the instructions in that read me. 

Can you please point me in the right direction to pin the `exception_cause_supported` branch in `rspec-support`?